src/config.c

```diff
@@ -648,6 +657,19 @@ short load_configuration(void)
                 COMMAND_TEXT(cmd_num),config_textname);
           }
           break;
+      case 10: // Atmospheric sound
+          i = recognize_conf_parameter(buf,&pos,len,logicval_type);
+          if (i <= 0)
+          {
+              CONFWRNLOG("Couldn't recognize \"%s\" command parameter in %s file.",
+                COMMAND_TEXT(cmd_num),config_textname);
```

Comment:
Wrong indent

Reply:
This one is copied exactly from the other settings listed above. What do you suggest? Changing all 10 settings?